pkg/await/apps_deployment.go

```diff
@@ -459,7 +458,7 @@ func (dia *deploymentInitAwaiter) checkReplicaSetStatus() {
 	// we would now have fewer replicas than we had requested in the `Deployment` we last submitted
 	// when we last ran `pulumi up`.
 	rawSpecReplicas, specReplicasExists := openapi.Pluck(rs.Object, "spec", "replicas")
-	specReplicas, _ := rawSpecReplicas.(float64)
+	specReplicas, _ := rawSpecReplicas.(int64)
```

Comment:
Hmm, looks like this is actually an `*int32`? Have we double checked the type with `reflect.TypeOf`, just to be safe?

Reply:
Just checked and it said `int64`